source/cloud-service/your-environment/settings.rst

```diff
@@ -0,0 +1,95 @@
+.. Copyright (C) 2015, Wazuh, Inc.
+
+.. meta::
+  :description: Learn about Wazuh Cloud settings. 
+
+.. _cloud_your_environment_settings:
+
+Settings
+========
+
+Every cloud environment is configured based on specific settings that define its limitations, capabilities, and pricing. We offer six available settings, including four basic settings and two advanced settings. The advanced settings are automatically calculated based on the basic settings but can be modified if desired.
+
+To monitor the behavior of your environment and check if the configured values of the settings are being reached, see the :doc:`Monitor usage section </cloud-service/your-environment/monitor-environment-usage>`. section.
+
+Understanding environment settings
+----------------------------------
+
+Below, we provide a detailed explanation of each setting.
+
+.. _cloud_settings_active_agents:
+
+Active agents
+^^^^^^^^^^^^^
+
+This basic setting defines the maximum number of active Wazuh agents that the environment can support. Please note that while it is possible to register an unlimited number of Wazuh agents, the number of active agents is limited by this setting.
+
+If the maximum number of active agents is reached, the environment may start to malfunction, causing instability with agent connections. While the system can tolerate temporarily exceeding the limit of active agents, appropriate measures will be taken if the situation persists.
+
+
+.. _cloud_settings_indexed_data:
+
+Indexed data
+^^^^^^^^^^^^
+
+The indexed data, previously known as hot storage, includes the data available on the Wazuh WUI, which corresponds to the information indexed by Wazuh. This information becomes searchable and analyzable as soon as Wazuh ingests and indexes the events sent by the agents.
+
+Two settings define the behavior of the indexed data:
+
+- Indexed data time: It determines the maximum duration for which data remains indexed. This is a basic setting.
+
+- Indexed data size: It defines the maximum size, in bytes, of the indexed data. This setting is equivalent to what was previously called a "tier." This is an advanced setting, and the interface provides a suggestion when selecting the Indexed data time.
```

Comment:
It's not quite clear what the "... suggestion when selecting the Indexed data time." is all about.